src/EPR.Calculator.Frontend.UnitTests/DashboardControllerTests.cs

```diff
@@ -41,7 +44,7 @@ public async Task DashboardController_Success_View_Test()
                 .Setup(_ => _.CreateClient(It.IsAny<string>()))
                 .Returns(httpClient);
 
-            var controller = new DashboardController(GetConfigurationValues(), mockHttpClientFactory.Object);
+            var controller = new DashboardController(ConfigurationItems.GetConfigurationValues(), mockHttpClientFactory.Object);
```

Comment:
Can we have a var created for `ConfigurationItems.GetConfigurationValues()`
and use it rather than calling the same method again?

Reply:
It is called in every test and not repeated in a single test. Other option will be to create a private member and initialise it in the constructor, but I think constructor will be called for every test run.

Reply:
I just checked, and the constructor is invoked for each and every test.

Reply:
Thanks Uday, I have updated the code now.